components/DropDownOption.tsx

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable react/react-in-jsx-scope */
```

Comment:
import React, { Component }  from 'react';

Reply:
https://stackoverflow.com/questions/42640636/react-must-be-in-scope-when-using-jsx-react-react-in-jsx-scope